index.html

```diff
@@ -400,58 +460,62 @@ <h4 class="no-margin"><strong>The World Bank</strong>&nbsp; <small>has a wealth
             <!-- /.row -->
         </div>
         <!-- /.container -->
-        </div>
+    </section>
 
 
-        <!-- Footer -->
-        <footer>
-            <div class="container">
-                <div class="row">
-                    <div class="col-lg-10 col-lg-offset-1 text-center">
-                        <p class="text-muted">CC-BY</p>
-                    </div>
+    <!-- Footer -->
+    <footer>
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-10 col-lg-offset-1 text-center">
+                    <p class="text-muted">CC-BY</p>
                 </div>
             </div>
-        </footer>
-
-
-        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
-        <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
-        <script src="//d3js.org/d3.v3.min.js" charset="utf-8"></script>
-        <script src="js/blueimp-gallery.min.js"></script>
-        <script src="js/sims.js" charset="utf-8"></script>
-
-        <script>
-
-            // Closes the sidebar menu
-            $("#menu-close").click(function (e) {
-                e.preventDefault();
-                $("#sidebar-wrapper").toggleClass("active");
-            });
-
-            // Opens the sidebar menu
-            $("#menu-toggle").click(function (e) {
-                e.preventDefault();
-                $("#sidebar-wrapper").toggleClass("active");
-            });
-
-            // Scrolls to the selected menu item on the page
-            $(function () {
-                $('a[href*=#]:not([href=#])').click(function () {
-                    if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '') || location.hostname == this.hostname) {
-
-                        var target = $(this.hash);
-                        target = target.length ? target : $('[name=' + this.hash.slice(1) + ']');
-                        if (target.length) {
-                            $('html,body').animate({
-                                scrollTop: target.offset().top
-                            }, 1000);
-                            return false;
-                        }
+        </div>
+    </footer>
+
+
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="js/d3.js"></script>
```

Comment:
lets move all vendor js and css into js/vendor/*.js

We might also want to start using bower to handle dependencies for us.